<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mashtab</class>
 <widget class="QWidget" name="Mashtab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDoubleSpinBox" name="Xshift">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="Yshift">
       <property name="minimum">
        <double>-1000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Смещения по X и Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="newH">
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="newW">
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Новые высота и ширина</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="Neighbor">
     <property name="text">
      <string>Ближфйший сосед</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="Bilineral">
     <property name="text">
      <string>Билинейное</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="Lanckhoz3">
     <property name="text">
      <string>Ланцоса3</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="BCsplain">
       <property name="text">
        <string>ВС-сплайн</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="Bcoef">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="Ccoef">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="show">
       <property name="text">
        <string>Посмотреть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sawe">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
